neo/SmartContract/InteropService.cs

```diff
@@ -143,8 +143,39 @@ private static bool Runtime_GetTrigger(ApplicationEngine engine)
 
         internal static bool CheckWitness(ApplicationEngine engine, UInt160 hash)
         {
-            var _hashes_for_verifying = engine.ScriptContainer.GetScriptHashesForVerifying(engine.Snapshot);
-            return _hashes_for_verifying.Contains(hash);
+            if (engine.ScriptContainer is Transaction tx)
+            {
+                CosignerUsage usage = tx.Attributes
```

Comment:
We should decide if we want to process the first or all of them

Reply:
https://github.com/neo-project/neo/blob/53f7d022afba7b1dfce5a437f8bc11557423a7fb/neo/Network/P2P/Payloads/Transaction.cs#L95-L96

In fact it is impossible.